fs/remote/resolver.go

```diff
@@ -73,7 +73,8 @@ const (
 	defaultValidIntervalSec = 60
 	defaultFetchTimeoutSec  = 300
 
-	defaultMaxRetries  = 5
+	// defaults based on a target total retry time of at least 5s. 30*((2^8)-1)>5000
+	defaultMaxRetries  = 8
```

Comment:
suggest to `7` which makes jitter in the range of `[3.8, 7.6]` (5.7 on average) and fits more into the expected 5s.

Reply:
@sbuckfelder requested "more than 5 seconds" in planning elsewhere. I interpreted this as a minimum of 5, not a target/average of 5.

~5-10s could be achieved with 8 retries and 20ms minimum, or 7 retries and 40ms minimum. However, changing the minimum could have side effects such as triggering throttling or worsening the average time to success.

Perhaps +100% is not a reasonable jitter range? If we reduce the jitter range then we could have 8-10s instead of 8-16s as the range, without additional changes. @djdongjin?

Reply:
I've changed the jitter calculation from +100% to +12.5% by adding a divisor to the jitter function. So the range is now ~8-9s.
Conversation remains open until we decide if this is still too long.

Reply:
Resolving based on approval from commenters.